adminapi/src/main/java/io/minio/admin/messages/info/DiskMetrics.java

```diff
@@ -40,6 +40,18 @@ public class DiskMetrics {
   @JsonProperty("totalErrorsTimeout")
   private Integer totalErrorsTimeout;
 
+  @JsonProperty("totalTokens")
```

Comment:
Please add `@JsonIgnoreProperties(ignoreUnknown = true)` to this class

Reply:
ok，I've added